source/CMakeLists.txt

```
@@ -23,11 +23,21 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+macro(add_pkgconfig_library)
+  if(PKGCONFIG_LIBRARIES)
+   set(PKGCONFIG_LIBRARIES "${PKGCONFIG_LIBRARIES}, ${ARGV0} >= ${ARGV1}")
+ else()
+   set(PKGCONFIG_LIBRARIES "${ARGV0} >= ${ARGV1}")
+ endif()
```

Indentation looks a bit off here
```suggestion
  if(PKGCONFIG_LIBRARIES)
   set(PKGCONFIG_LIBRARIES "${PKGCONFIG_LIBRARIES}, ${ARGV0} >= ${ARGV1}")
  else()
    set(PKGCONFIG_LIBRARIES "${ARGV0} >= ${ARGV1}")
  endif()
```